en/flight_modes/return.md

```diff
@@ -15,26 +15,52 @@ The specific behaviour for each vehicle type is described below.
 
-> **Note** The `RTL_LAND_DELAY` is provided to allow time for landing gear to be deployed (this is triggered automatically). By default this period is short so that the vehicle will simply slow and then land immediately. The parameter can also be set so that the vehicle will hover indefinitely.
 
 The RTL behaviour can be configured using the parameters below.
 
 
 Parameter | Description
 --- | ---
-<span id="RTL_RETURN_ALT"></span>[RTL_RETURN_ALT](../advanced_config/parameter_reference.md#RTL_RETURN_ALT) | Minimum RTL return altitude in meters (default: 60m). The vehicle will ascend to this altitude before returning. If already above this value the vehicle will return at its current altitude.
-<span id="RTL_DESCEND_ALT"></span>[RTL_DESCEND_ALT](../advanced_config/parameter_reference.md#RTL_DESCEND_ALT) | Altitude at which the vehicle will slow or stop its initial descent (default: 30m)
-<span id="RTL_LAND_DELAY"></span>[RTL_LAND_DELAY](../advanced_config/parameter_reference.md#RTL_LAND_DELAY) | Time to hover at `RTL_DESCEND_ALT` before landing (default: 0.5s). If set to -1 the system will loiter at `RTL_DESCEND_ALT` rather than landing.
-<span id="RTL_MIN_DIST"></span>[RTL_MIN_DIST](../advanced_config/parameter_reference.md#RTL_MIN_DIST) | Minimum horizontal distance from home position to trigger ascent to a safe altitude (RTL_RETURN_ALT). If the vehicle is horizontally closer than this distance to home, it will return at its current altitude (instead of first ascending to RTL_RETURN_ALT). 
+<span id="RTL_RETURN_ALT"></span>[RTL_RETURN_ALT](../advanced_config/parameter_reference.md#RTL_RETURN_ALT) | Return altitude in meters (default: 60m) when [RTL_CONE_ANG](../advanced_config/parameter_reference.md#RTL_CONE_ANG) is 0. If already above this value the vehicle will return at its current altitude.
+<span id="RTL_DESCEND_ALT"></span>[RTL_DESCEND_ALT](../advanced_config/parameter_reference.md#RTL_DESCEND_ALT) | Minimum return altitude and altitude at which the vehicle will slow or stop its initial descent from a higher return altitude (default: 30m)
+<span id="RTL_LAND_DELAY"></span>[RTL_LAND_DELAY](../advanced_config/parameter_reference.md#RTL_LAND_DELAY) | Time to hover at `RTL_DESCEND_ALT` before landing (default: 0.5s). If set to -1 the system will loiter at `RTL_DESCEND_ALT` rather than landing. This is provided to allow time for landing gear to be deployed (triggered automatically on MC).
+  By default this period is short so that the vehicle will simply slow and then land immediately.
+  The parameter can also be set so that the vehicle will hover indefinitely.
+<span id="RTL_MIN_DIST"></span>[RTL_MIN_DIST](../advanced_config/parameter_reference.md#RTL_MIN_DIST) | Minimum horizontal distance from home position to trigger ascent to a safe altitude (RTL_RETURN_ALT). If the vehicle is horizontally closer than this distance to home, it will return at its current altitude (instead of first ascending to RTL_RETURN_ALT).

```

Comment:
Does this still work - ie is minimum ascent to RTL_DESCEND_ALT ignored within RTL_MIN_DIST?

Reply:
@hamishwillee This is almost correct. The vehicle will ALWAYS ascend to  RTL_DESCEND_ALT even if it's within RTL_MIN_DIST of the home position.

Reply:
Thanks. Fixed.